package.json

```diff
@@ -70,6 +70,7 @@
     "typescript": "2.1.4"
   },
   "dependencies": {
+    "form-data": "^2.5.0",
```

Comment:
why do we need this?

Reply:
Not sure why, but without this dependence build was failed.